anchor-reference-server/src/main/java/org/stellar/anchor/reference/controller/CustomerController.java

```diff
@@ -0,0 +1,57 @@
+package org.stellar.anchor.reference.controller;
+
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestMethod;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.RestController;
+import org.stellar.anchor.exception.SepNotFoundException;
+import org.stellar.anchor.integration.customer.*;
+import org.stellar.anchor.reference.config.AppSettings;
+import org.stellar.anchor.reference.service.CustomerService;
+
+@RestController
+public class CustomerController {
```

Comment:
I just want confirm my understanding before I deep dive into this PR.

This `CustomerController` class is the server-side implementation of the requests made from the platform right?

In that case, this controller should use the endpoints and request / response schemas defined here: https://github.com/stellar/stellar-anchor-platform/blob/main/design/callbacksApi/api.yml#L268

From what I can see now, we're using the `/customers` path, but the Callback API uses `/customer`. Also, I see we're using the `integration.customer` request & response classes here, but those classes don't match the schemas used in the Callback API. For example, the Callback API includes the `lang` parameter.

Reply:
> I just want confirm my understanding before I deep dive into this PR.

This PR has rough edges. It is not necessary to do a full review at this moment yet. I will continue to implement end-to-end sep-12 implementation. Then we can do a full review. 

> 
> This `CustomerController` class is the server-side implementation of the requests made from the platform right?
> 
Yes. 

> In that case, this controller should use the endpoints and request / response schemas defined here: https://github.com/stellar/stellar-anchor-platform/blob/main/design/callbacksApi/api.yml#L268
Yes, that's right. Can you push commits to the `develop-12` branch? 

> 
> From what I can see now, we're using the `/customers` path, but the Callback API uses `/customer`. Also, I see we're using the `integration.customer` request & response classes here, but those classes don't match the schemas used in the Callback API. For example, the Callback API includes the `lang` parameter.

This PR is a starting point. We should make it conform to the design. https://github.com/stellar/stellar-anchor-platform/blob/main/design/callbacksApi/api.yml#L268

Reply:
Once I create the `platform-apis` subproject, the reference controller can import those classes?